.github/workflows/go.yml

```diff
@@ -3,9 +3,9 @@ on:
   push:
     branches:
       - master
-  pull_request:
-    branches:
-      - master
+  # pull_request:
```

Comment:
please remove these lines (if needed we can add them again later)

Reply:
removed